operator.yml

```diff
@@ -0,0 +1,325 @@
+---
```

Comment:
It is unnecessary to create an entirely new playbook, especially when > 95% of the code is identical.

Use of either tags, or a boolean var should be all that is necessary to select the correct tasks.

Reply:
Hi @bcrochet , agreed. I just added the fips_assessment tags. All we need to do is run the audits playbook with --skip-tags fips_assessment.